pkg/config/config.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+type Config struct {
+	maskSessionPII bool
```

Comment:
Not sure if there is a better place to put this config. I initially tried putting this inside the logger package, however I ran into circular dependency issues when trying to import it from the `session_state.go` file